apps/hash-frontend/src/pages/[shortname]/[page-slug].page.tsx

```diff
@@ -393,10 +393,10 @@ const Page: NextPageWithLayout<PageProps> = ({
 
       <PageContextProvider pageEntityId={pageEntityId}>
         <Box
-          sx={({ zIndex, palette }) => ({
+          sx={({ palette }) => ({
             position: "sticky",
             top: 0,
-            zIndex: zIndex.appBar,
+            zIndex: 10,
```

Comment:
We changed this from something connected to something else, to something independent – can we connect it back to something else? maybe `appBar + 1`? what is it fighting with?

Reply:
Addressed here https://github.com/hashintel/hash/pull/1809/commits/916515b842b88016061425376e0fa7d75c5290fb. Removed a wrongly defined z-index from MUI theme options.